INSTALL.md

```diff
@@ -84,9 +84,20 @@ Dangerzone is available for:
   </tr>
 </table>
 
-Add our repository following these instructions:
+#### Retrieve the PGP keys
```

Comment:
Did you plan to add a section as well for installing Dangerzone? Else, maybe we can use normal text here.

Reply:
Thanks, updated in c52da93c90ce4d9ca9bf7eb7a2c8e47e6c77ac17